examples/mysql-client/mysql_client.bal

```diff
@@ -0,0 +1,79 @@
+import ballerina/io;
+import ballerina/log;
+import ballerina/mysql;
+import ballerina/sql;
+
+endpoint mysql:Client testDB {
+    host: "localhost",
+    port: 3306,
+    name: "testdb",
+    username: "root",
+    password: "root",
+    poolOptions: {maximumPoolSize: 5}
+};
+
+function main(string... args) {
+
+    // Create a DB table using the `update` action. If the DDL
```

Comment:
We need to mention refer jdbc_client sample for more details on supported functions

Reply:
This was added to the description.